README.md

```diff
@@ -0,0 +1,58 @@
+# Flatpak for DOSBox
+
+## Installation
+
+This Flatpak is available on
+[Flathub](https://flathub.org/apps/details/com.dosbox.DOSBox).
+After following the [Flatpak setup guide](https://flatpak.org/setup/),
+you can install it by entering the following command in a terminal:
+
+```bash
+flatpak install flathub com.dosbox.DOSBox -y
+```
+
+Once the Flatpak is installed, you can run DOSBox using your desktop environment's
+application launcher.
```

Comment:
Adding "or by running  `flatpak run com.dosbox.DOSBox` in a terminal." could be a good idea since we also have the other things as terminal commands here.

Reply:
I will merge the PR once this is resolved.